platform/how-to/use-go-template.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Use Go Template Functions
+sidebar_label: Use Go Template Functions
+sidebar_position: 9
+description: Learn how to use Go template functions in vCluster Platform templates.
+---
+### Overview
+Templates in vCluster Platform standardize configurations for virtual clusters, namespaces, and applications.
+You can create templates through the vCluster Platform UI or by applying manifests directly to your cluster.
+The platform provides the VirtualClusterTemplate CRD for defining templates.
+For more information, see [Creating Templates](../administer/templates/create-templates).
+
+<!-- vale off -->
+### Use Go template functions
+<!-- vale on -->
+When defining a VirtualClusterTemplate CRD, you can use Go template functions in the same way they are used in Helm charts. These functions enable conditional logic and dynamic configuration within your template values, allowing you to customize deployments based on context or user input.
+
+```yaml title="Use eq in VirtualClusterTemplate CRD Definition"
+apiVersion: management.loft.sh/v1
+kind: VirtualClusterTemplate
+metadata:
+  name: vcluster-pro-template
+spec:
+  displayName: Virtual Cluster Pro Template
+  description: This virtual cluster template deploys a vCluster Pro
+  template:
+    metadata: {}
+    instanceTemplate:
+      metadata: {}
+    apps:
+      - name: helm-dashboard
+        namespace: helm-dashboard
+    pro:
+      enabled: true
+    helmRelease:
+      chart:
+        version: 0.20.0
+      values: >-
+        # Below you can configure the virtual cluster
+        controlPlane:
+          coredns:
+            embedded: true
+        sync:
+          fromHost:
+            ingressClasses:
+              enabled: true
+          toHost:
+            ingresses:
+              {{ if eq .Values.loft.project "auth-core" }}
```

Comment:
This is a nice example of using conditionals, but I think it would be nice to have a more comprehensive set of examples - they could all be part of this same template - to include:

- Using template parameter values
- Using the vCluster name as maybe a dynamic ingress host - https://github.com/loft-demos/vcluster-platform-demo-app-template/blob/main/vcluster-gitops/virtual-cluster-templates/base/default.yaml#L43
- Using a connected host cluster parameter - https://www.vcluster.com/docs/platform/administer/templates/parameters#connected-host-cluster-parameter-values - example for sleep mode: https://github.com/loft-demos/vcluster-platform-demo-app-template/blob/main/vcluster-gitops/virtual-cluster-templates/base/default.yaml#L61